src/browser/browser.ts

```diff
@@ -357,6 +362,8 @@ export class Browser {
       this.log.error('Error while waiting for the panels to load', 'url', options.url, 'err', err.stack);
     }
 
+    await new Promise(r => setTimeout(r, options.fullPageImageDelayTime!! * 1000));
```

Comment:
I think the pull request looks generally good, but this line made me realize the unit used to express the delay is explained nowhere. 

So, what do you think about slightly changing its name by `fullPageImageDelaySeconds` or at least document somewhere that the unit for that setting are seconds?

Thanks!